ui-speedspacechart/src/stories/assets/sampleData.ts

```diff
@@ -33137,6 +33137,22 @@ export const SAMPLE_DATA: OsrdSimulationState = {
       stop: 201408.607,
     },
   ],
+
+  speedLimitTags: {
+    boundaries: [0, 25000, 65000, 95000, 125000, 201408.607],
+
+    values: [
+      { speed_limit_tags_type: 'tag', tag_name: 'MA100', color: '#494641' },
+      { speed_limit_tags_type: 'tag', tag_name: 'EVO', color: '#216482' },
+      { speed_limit_tags_type: 'tag', tag_name: 'incompatible', color: '#EAA72B' },
+      {
+        speed_limit_tags_type: 'tag',
+        tag_name: 'UU',
```

Comment:
Sketch : 
![Capture d’écran 2024-07-12 à 11 48 01](https://github.com/user-attachments/assets/ab5001c8-eb2f-4454-9f9b-8f9fcb2be03f)

PR : 
![Capture d’écran 2024-07-12 à 11 49 19](https://github.com/user-attachments/assets/5d631968-ee2a-4567-a10d-64dbb0fcfe03)

There should be a dark border around the tag name (it's probably present in other tags too but can't be seen because of the colors contrast)

Reply:
done